tests/flags.go

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017 Red Hat, Inc.
+ *
+ */
+
+package tests
+
+import (
+	"flag"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+var KubeVirtUtilityVersionTag = ""
```

Comment:
Is there a reason why you are not using a struct for this?

Reply:
- In order not to change each user of it at this time.
- These are also used independently all over and not as a group, so I see no advantage to change it at the moment. But maybe there are groups in this big list which do merit placing into structs.

Reply:
I was just wondering whether we could improve on the structure of all this, but I don't want to put this burden on you. Maybe we can do this in a later step.